net/openssh/Makefile

```diff
@@ -8,13 +8,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openssh
-PKG_VERSION:=9.8p1
+PKG_REALVERSION:=9.8p1
+PKG_VERSION:=9.8_p1
```

Comment:
PKG_VERSION:=$(subst p,_p,$(PKG_REALVERSION))

Reply:
That was my original approach in e655193,   but CI rejected it, so I moved to manual strategy to avoid CI trouble in future...

Reply:
hmm? That should evaluate to the same thing.